RecoPixelVertexing/PixelTriplets/interface/gpuPixelDoublets.h

```diff
@@ -9,9 +9,9 @@
 #include <limits>
 
 #include "DataFormats/Math/interface/approx_atan2.h"
-#include "RecoLocalTracker/SiPixelRecHits/plugins/siPixelRecHitsHeterogeneousProduct.h"
+#include "RecoLocalTracker/SiPixelRecHits/interface/siPixelRecHitsHeterogeneousProduct.h"
 
-#include "GPUCACell.h"
+#include "RecoPixelVertexing/PixelTriplets/interface/GPUCACell.h"
```

Comment:
Also here forward-declaring `GPUCACell` would be sufficient.

`gpuPixelDoublets.h` itself seems not be used outside `RecoPixelVertexing/PixelTriplets/src`, so I'd move it to the `src`.